src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -1080,7 +1082,23 @@ private final class GraphL implements GraphListener {
      */
     private final class NodePrintListener implements GraphListener.Synchronous {
         @Override public void onNewHead(FlowNode node) {
-            NewNodeConsoleNote.print(node, getListener());
+            // TODO: Breaks show/hide links in the console because the line with the show/hide link gets hidden.
```

Comment:
Should be fixable, but I am not going to bother looking into it unless I really want to get this merged.